vtex-sites/nextjs.store
# Feat: Replaces `DiscountBadge` FSUI

## What's the purpose of this pull request?

- Uses `DiscountBadge` from the `faststore/ui`
- Adds some adjusts to it work properly
- Remove `Badge` folder

## How does it work?

Everything should look as before. (Visually)

## How to test it?

- You can check the components that use `DiscountBadge` on the storybook , such as [ProductCard](https://nextjs-store-storybook-git-chore-replace-disco-87d50d-faststore.vercel.app/?path=/docs/molecules-productcard--overview-default) and [ProductTitle](https://nextjs-store-storybook-git-chore-replace-disco-87d50d-faststore.vercel.app/?path=/docs/molecules-producttitle--default-story)

## References

Should be merged when https://github.com/vtex/faststore/pull/1546 is available.
